website/blog/2024-02-12-How-a-POC-became-a-production-ready-Hudi-data-lakehouse-through-close-team-collaboration.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: "How a POC became a production-ready Hudi data lakehouse through close team collaboration"
+excerpt: "How a POC became a production-ready Hudi data lakehouse through close team collaboration"
+author: leboncoin tech blog
```

Comment:
Author Info obtained from the blog :
`By Xiaoxiao Rey, Data Engineer, and [Hussein Awala](https://medium.com/@hussein-awala), Senior Data Engineer`

Can we use these author names?